lib/collector/serverless.js

```diff
@@ -36,7 +36,8 @@ class ServerlessCollector {
       function_version: null,
       execution_environment: process.env.AWS_EXECUTION_ENV,
       protocol_version: 16,
-      agent_version: agent.version
+      agent_version: agent.version,
+      agent_language: 'nodejs'
```

Comment:
We're using v1 of the payload, do you also have to update that constant at top of file?

Reply:
n/m, you answered in slack

Reply:
We shouldn't update the constant: v2 payloads have a slightly different structure, and use different endpoints.